src/ModAlgAss/Lattices/Morphisms.jl

```diff
@@ -254,6 +254,7 @@ end
 ################################################################################
 
 function _is_isomorphic_with_isomorphism_same_ambient_module(L::ModAlgAssLat, M::ModAlgAssLat, with_iso::Type{Val{T}} = Val{true}) where {T}
+  @info "Computing hom space as ideal"
```

Comment:
Maybe here and line 271 it would be better a `@vprintln`, or is `@info` kept on purpose for warning the users ?